app/konnect/analytics/reference.md

```diff
@@ -11,9 +11,11 @@ Metric | Description
 -------|------------
 Request Count | Total number of API calls within the selected time frame. This includes requests that were rejected due to rate limiting, failed authentication, and so on.
 Requests per Minute | Number of API calls per minute within the selected time frame.
-Response Latency | The amount of time, in milliseconds, that it takes to process an API request. The time starts when {{site.base_gateway}} receives a request and ends when it forwards the response back to the original caller. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response latency of 10 milliseconds means that every 1 in 100 requests took at least 10 milliseconds from request received until response returned. 
+Response Latency | The amount of time, in milliseconds, that it takes to process an API request. Users can select between average (avg) or different percentiles (p99, p95, and p50). For example, a 99th percentile response latency of 10 milliseconds means that every 1 in 100 requests took at least 10 milliseconds from request received until response returned. 
 Request Size | The size of the request payload received from the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.
 Response Size | The size of the response payload returned to the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response size of 100 bytes means that the payload size for every 1 in 100 response back to the original caller was at least 100 bytes.
+Upstream Latency | The amount of time taken, measured in milliseconds, for {{site.base_gateway}} to wait for the first byte of the upstream service response. in milliseconds, that Gateway was waiting for the first byte of the upstream service response. Users can select between different percentiles (p99, p95, and p50). For example, 99th percentile response latency of 10ms = 1 in 100 requests took >=10ms from request to upstream service until response.
```

Comment:
Any reason why the examples in both upstream and kong latency are different from the others? 

> For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.

vs

> For example, 99th percentile response latency of 10ms = 1 in 100 requests took >=10ms from request to upstream service until response.

Can we modify one or the other to stay consistent in how we explain percentiles, pls.

Reply:
Added